.gitignore

```diff
@@ -286,3 +286,4 @@ __pycache__/
 *.btm.cs
 *.odx.cs
 *.xsd.cs
+/glTF-Toolkit.Test/Resources/gltf
```

Comment:
Why? If we remove the resources we can't run the tests

Reply:
Must have gotten added automatically somewhere down the line, didn't intend for this change. Will fix.

Reply:
Removed.